frontend-react/src/components/Table/__snapshots__/Pagination.test.tsx.snap

```diff
@@ -47,7 +47,9 @@ exports[`Pagination Renders the expected markup when the current page is between
       >
         <button
           aria-label="Page 1"
-          class="usa-pagination__button rs-pagination-no-button"
```

Comment:
This is the _only_ snapshot test I know of. A former engineer used them during their brief time here, but we've never found them useful. In fact, this created a pain when updating tests because it's not in line with our standard testing methods.